### PR TITLE
Homework Lessons 24 and 25 (Functions and Objects)

### PR DESCRIPTION
**Pull request by FedkivMari**

### CODE REVIEW

File: Lesson25/js/first.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/* Create an object that describes a car (manufacturer, model, year of release, average speed)
+   and the following methods for working with this object:
+A method that displays the car info
```
Comment (ermiahp) [inline Lesson25/js/first.js:3]: Create a methods
U did not create any

File: Lesson24/assets/js/fourth.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* Write a function that accepts the width and length of a rectangle and calculates its area.
+ If there is only one parameter given, it counts as a square. */
+let area = 0;
+function caclArea(sideOne, sideTwo){
+     if (sideTwo==undefined)
```
[the remark this answers is not in the document]
Comment (ermiahp) [reply Lesson24/assets/js/fourth.js:5]: ===

File: Lesson24/assets/js/fifth.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    perfect number is a number equal to the sum of all its divisors.
+    For example, the smallest perfect number is 6, which is the sum of 1, 2, and 3. 
+   6-th: Write a function that accepts the min and max values of a range, 
+    and only puts out the perfect numbers within that range.
+    Use the previous function to check, if numbers are perfect.
+ */
+
+ 
+ let number =parseInt(prompt(`Enter a number`));
+ let res = checkPerfectNumber(number);
```
Comment (ermiahp) [inline Lesson24/assets/js/fifth.js:11]: u can combine this 2 lines

File: Lesson24/assets/js/fifth.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+ }
+
+ console.log(checkResult);
+ //----------------------------------------------------------
+
+ function checkPerfectNumber(num){
+    let sum = 0;
+    for (let i=1; i<num; i++)
+        {
+           if (num%i==0){
```
Comment (ermiahp) [inline Lesson24/assets/js/fifth.js:41]: 3 =

File: Lesson24/assets/js/fifth.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            sum+=i;
+           } 
+        } 
+    let res=((sum===num)? true: false);
```
[the remark this answers is not in the document]
Comment (FedkivMari) [reply Lesson24/assets/js/fifth.js:45]: Sorry, but I don't understand what you mean here. Can you write?

**Pull request merged**